examples/03_evaluate/als_movielens_diversity_metrics.ipynb

```diff
@@ -36,13 +36,16 @@
     "We evaluate the results obtained with two approaches: using the ALS recommender algorithm vs. a baseline of random recommendations. \n",
```

Comment:
<p>The above warning is strange, because there are 6 columns in the data frame, not 4. Any idea what happened? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
> The above warning is strange, because there are 6 columns in the data frame, not 4. Any idea what happened?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/microsoft/recommenders/pull/1505/discussion/)_

fixed schema definition. Thanks for catching it!